src/routes/(user)/clients/+page.svelte

```diff
@@ -3,6 +3,11 @@
 
 	export let data: PageData;
 
+	const formatTimestamp = (timestamp: string) => {
+		const data = new Date(timestamp);
+		return data.toLocaleString();
```

Comment:
Let's print the date as ISO8601 for now, as we are now mainly focussing on API routes. This can be done using the .toISOString() method.